install_config/storage_examples/ceph_example.adoc

```diff
@@ -273,3 +273,44 @@ spec:
 <1> `*securityContext*` must be defined at the pod level, not under a specific container.
 <2> All containers in the pod will have the same `*fsGroup*` ID.
 ====
+
+[[using-ceph-rbd-setting-default-secret]]
+== Setting ceph-user-secret as Default for Projects
+If you would like to make the persistant storage available to every project you have to modify the default project template.
+You can read more on modifying the default project template here link:https://docs.openshift.org/latest/admin_guide/managing_projects.html#selfprovisioning-projects[Self-provisioning Projects].
```

Comment:
Revise to:

Read more on xref:../admin_guide/managing_projects.adoc#selfprovisioning-projects[modifying the default project template].

Note the formatting of the link per our [doc guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references).